packages/amplication-gpt-gateway/src/kafka/kafka.controller.ts

```diff
@@ -1,14 +1,19 @@
 import { EventPattern, Payload } from "@nestjs/microservices";
 import { KafkaMessage } from "./KafkaMessage";
 import { Controller } from "@nestjs/common";
+import { MessageInput } from "../dto/MessageInput";
+import { ConversationTypeService } from "../conversationType/conversationType.service";
 
 @Controller("kafka-controller")
 export class KafkaController {
+  constructor(protected readonly conversationType: ConversationTypeService) {}
+
   @EventPattern("triggergptchatcompletion")
   async onTriggergptchatcompletion(
```

Comment:
Update the Topic name on amplication to GptConversationStart (and  GptConversationComplete for the other side)